src/main/java/com/ibm/crypto/plus/provider/CurveUtil.java

```diff
@@ -21,38 +20,31 @@ public enum CURVE {
         X25519, X448, FFDHE2048, FFDHE3072, FFDHE4096, FFDHE6144, FFDHE8192, Ed25519, Ed448
     }
 
-    private static HashMap<CURVE, Integer> publicCurveSizes = new HashMap<CURVE, Integer>(); // key sizes of curves in bytes
-    private static HashMap<Integer, CURVE> sizesToPublicCurves = new HashMap<Integer, CURVE>(); // reverse of above
-    private static HashMap<CURVE, Integer> privateCurveSizes = new HashMap<CURVE, Integer>(); // key sizes of curves in bytes
-    private static HashMap<Integer, CURVE> sizesToPrivateCurves = new HashMap<Integer, CURVE>(); // reverse of above
+    private static HashMap<CURVE, Integer> curveSizes = new HashMap<CURVE, Integer>(); // key sizes of curves in bytes
```

Comment:
Should we change the definition to `Map`?

Reply:
Looking up Map it appears that it is a interface not an implementation that we could use to store these values. I kept HashMap since we were using these before and i didnt see too much of a downside there.

Reply:
I meant something like `private static Map<CURVE, Integer> curveSizes = new HashMap<CURVE, Integer>();`

Reply:
Id prefer to use HashMap as its more explicitly defined unless there is a reason I am not aware of?

Reply:
It's just common practice to declare the variable with the interface and use the concrete class for the initialization. But I'm good either way.

Reply:
OK good point here. Will update to use `Map` in the delcarations of our `HashMap` values.

Reply:
Done